xcode/Subconscious/SubconsciousTests/Tests_DataService.swift

```diff
@@ -163,13 +158,11 @@ final class Tests_DataService: XCTestCase {
         
         let detail = try data.readDetail(
             address: address,
-            title: "Test fallback",
             fallback: "Fallback content"
         )
         
         XCTAssertEqual(detail.entry.address, address)
         XCTAssertEqual(detail.entry.contents.body, memo.body)
-        XCTAssertEqual(detail.entry.contents.title, memo.title)
     }
     
     func testManyWritesThenCloseThenReopen() throws {
```

Comment:
This test did not change in this PR, but it fails for me locally... Can you confirm if it passes for you?

`testManyWritesThenCloseThenReopen(): XCTAssertEqual failed: ("bafy2bzacecrxijbvgzo3shfqmzfk6wazsyovevdsuur6zhvbtzfz752m2hqqi") is not equal to ("bafy2bzacebu4uahwwzbz3n3jzz2wvos2oypodx2msxasgqpowkzu332uv5qe6")`

`testManyWritesThenCloseThenReopen(): failed: caught error: "fileDoesNotExist("/b")"`

Reply:
@bfollington it's from a previous PR. Was originally added to troubleshoot the persistence issue we had with Noosphere earlier. Fails because Swift destructors are not guaranteed to run until end of scope. Was testing to see if this was causing the issue. It does cause _a_ issue (hence failing test). However, it was not _the_ issue.

I think we can remove it now that Noosphere flushes on save, or rewrite it so that it passes in this state.

Reply:
Follow-up here https://github.com/subconsciousnetwork/subconscious/pull/441